components/automate-gateway/handler/iam/v2/introspect/introspect_test.go

```diff
@@ -257,7 +256,7 @@ func testServerAndHandler(t *testing.T) (
 
 	return authSrv,
 		s,
-		handler.NewAuthzServer(
+		handler_introspect.NewServer(
```

Comment:
nit: Since you've renamed `handler` -> `introspect`, I was confused about `handler` reappearing here. It makes sense to either nix it or use it in both places.